python/pyspark/mllib/tree.py

```diff
@@ -90,9 +92,11 @@ def predict(self, x):
         else:
             return self.call("predict", _convert_to_vector(x))
 
+    @property
```

Comment:
I agree.  Can you please change the doc test back to what it was to make sure we don't break APIs?  If we can't support these as properties, that is OK.